### DISCUSSION TITLE
Filter according to the optionGroupLabel name in multi-seelct

### DISCUSSION THREAD

**Discussion by Uzeyir9695** [primefaces/community#519] [PrimeVue] [answered]
Is it possible to filter according to the optionGroupLabel name in multi-seelct instead of option name? BY default it filters according to the option name.

**Comment by tugcekucukoglu** [comment, marked as answer]
As you said, MultiSelect filters according to optionLabel. optionGroupLabel is thought of as a kind of group header. If we look at it in terms of accessibility, it is not an element that receives focus. However, if different libraries have done similar work, we can include it in the roadmap.
